src/docfx/lib/JsonUtility.cs

```diff
@@ -213,6 +237,25 @@ private static void LogWarningForNullValue(JToken item, List<Error> errors, JTok
             }
         }
 
+        private static List<Error> RegisterLicense(string license)
+        {
+            var errors = new List<Error>();
```

Comment:
True, it can only be one, got used to return a collection of errors everywhere

---
In reply to: [201892445](https://github.com/dotnet/docfx/pull/3052#discussion_r201892445) [](ancestors = 201892445)

Reply:
No need, it's based on your implementation and requirement.

---
In reply to: [201894021](https://github.com/dotnet/docfx/pull/3052#discussion_r201894021) [](ancestors = 201894021,201892445)